docs/mobile-apps/automated-testing/espresso-xcuitest/espresso.md

```diff
@@ -1296,25 +1296,12 @@ Application settings for real device tests.
 
 ```yaml
 appSettings:
-  audioCapture: true
   instrumentation:
     networkCapture: true
 ```
 
 ---
 
-#### `audioCapture`
```

Comment:
Shouldn't this remain? Real devices are still supported after all, no? Same for XCUITest on RDC.